### ISSUE TITLE
wood outside should become wet after it rains - realism

### ISSUE THREAD

**Issue by Arne18** [CleverRaven/Cataclysm-DDA#27773]
**Is your feature request related to a problem? Please describe.**  
n/a

**Describe the solution you'd like**  
make logs/wood become wet after it rains outside

**Describe alternatives you've considered**  
n/a

**Additional context**  
n/a

**Comment by Arne18** [comment]
Obviously, It will challenge you in that way it will get harder to maintain a fire. Just a small suggestion.

**Comment by Barhandar** [comment]
Pretty sure the game does not model all the ways you could keep firewood dry (putting it under a pine tree, placing sheets over it, even just stacking it so only top logs/planks/whatever get wet) and as such the only thing this will do is being annoying. It does not challenge players in any way.

**Comment by ZhilkinSerg** [comment]
Somehow I've read "wool" instead of "wood" - and that reminds me that soaking would be nice to have too.
